quickwit/quickwit-indexing/src/actors/merge_executor.rs

```diff
@@ -219,78 +221,6 @@ pub fn combine_partition_ids(splits: &[SplitMetadata]) -> u64 {
     combine_partition_ids_aux(splits.iter().map(|split| split.partition_id))
 }
 
-async fn merge_split_directories(
```

Comment:
this just became a method. Diff is not very good at that, but the changed is trivial.